cibyl/cli/argument.py

```diff
@@ -64,11 +64,11 @@ def parse_ranges(self, expressions):
             parsed_expressions.append(Range(operator, operand))
         return parsed_expressions
 
-    def __bool__(self):
+    def __bool__(self) -> bool:
         return bool(self.value)
 
-    def __eq__(self, other):
+    def __eq__(self, other: 'Argument') -> bool:
```

Comment:
There is no need to type hint builtin functions. These are standard and well known enough that people already know what they are dealing with. Besides, hints will not help much as these are rarely called explicitly. For example, equality is called from a statement like: A == B and Python will not care about their types in such case.